### PR TITLE
Backend,Tests,Frontend: use FSharpUtils from Fsdk

### PR DESCRIPTION
**Pull request by webwarrior-ws**
Added Fsdk package. Replaced FSharpUtils module by one from Fsdk. Removed tests that were moved to Fsdk.

### CODE REVIEW

**Comment by knocte** [comment]
Please also remove ConsoleReadPasswordLine() from UserInteraction.fs to use the one from Fsdk.

File: src/GWallet.Backend/FSharpUtil.fs [hunk 1]
```diff
@@ -78,20 +74,8 @@ module FSharpUtil =
...
 
     module AsyncExtensions =
-        let private makeBoxed (job: Async<'a>) : Async<obj> = 
-            async { 
-                let! result = job
-                return box result
-            }
-
         // FIXME: remove once we can stop supporting old versions of FSharpCore
         let SleepSpan (span: TimeSpan) =
```
Comment (knocte) [inline src/GWallet.Backend/FSharpUtil.fs:80]: @webwarrior-ws why didn't you move SleepSpan to Fsdk?

Comment (webwarrior-ws) [reply src/GWallet.Backend/FSharpUtil.fs:80]: Because of `#if LEGACY_FRAMEWORK` conditional compilation.

Comment (knocte) [reply src/GWallet.Backend/FSharpUtil.fs:80]: Is that really an issue? If Async.Sleep(TimeSpan) overload is not included in FSharp.Core 4.7.0 then you can just leave the alternative call, which works in both.

Comment (webwarrior-ws) [reply src/GWallet.Backend/FSharpUtil.fs:80]: Should I open a  new PR with `SleepSpan` in fsx?

Comment (knocte) [reply src/GWallet.Backend/FSharpUtil.fs:80]: just include the commit in the TSV PR

Comment (knocte) [reply src/GWallet.Backend/FSharpUtil.fs:80]: let's update this PR now to use Fsdk's SleepSpan

**Pull request merged**